ccpp/suites/suite_FV3_GFS_v15p2_ras.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<suite name="FV3_GFS_v15p2_ras" version="1">
```

Comment:
Is it possible to create a suite_FV3_GFS_v16_ras.xml suite file? The operational GFSv16 will be implemented this week, and we hope to update global suite files to be GFS_v16 based.

Reply:
I replaced `ccpp/suites/suite_FV3_GFS_v15p2_ras.xml` with `ccpp/suites/suite_FV3_GFS_v16_ras.xml` and also changed the regression tests in the ufs-weather-model PR.